src/suggestion-box.ts

```diff
@@ -0,0 +1,383 @@
+import { LitElement, html, css } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import { queryAssignedElements } from "lit/decorators.js";
+
+@customElement("aalam-sgn-box")
+export class SuggestionBox extends LitElement {
+    static styles = css`
+        :host {
+            display: block;
+            position: relative;
+        }
+    `;
+
+    @property({ type: Array<Object> })
+    list: Array<{ [key: string]: any }> = [];
+
+    @property()
+    listkey: string = "";
+
+    @property()
+    slot_input = null;
+
+    @property()
+    _private_slot_el: boolean = false;
```

Comment:
There seems to be no use for this variable. You are using it in set suggestions where you are just setting it to true in a loop. Even if its not there, the functionality will work.